corehq/apps/accounting/management/commands/cchq_software_plan_bootstrap.py

```diff
@@ -27,118 +27,115 @@
     SoftwarePlanEdition.ADVANCED: 'advanced_plan_v0',
     SoftwarePlanEdition.ENTERPRISE: 'enterprise_plan_v0',
 }
-EDITIONS = [
-    SoftwarePlanEdition.COMMUNITY,
-    SoftwarePlanEdition.STANDARD,
-    SoftwarePlanEdition.PRO,
-    SoftwarePlanEdition.ADVANCED,
-    SoftwarePlanEdition.ENTERPRISE,
+
+FEATURE_TYPES = [
+    FeatureType.USER,
+    FeatureType.SMS,
+]
+PRODUCT_TYPES = [
+    SoftwareProductType.COMMCARE,
+    SoftwareProductType.COMMCONNECT,
+    SoftwareProductType.COMMTRACK,
 ]
-FEATURE_TYPES = [f[0] for f in FeatureType.CHOICES]
-PRODUCT_TYPES = [p[0] for p in SoftwareProductType.CHOICES]
+
+BOOTSTRAP_PRODUCT_RATES = {
+    SoftwarePlanEdition.COMMUNITY: dict(),
+    SoftwarePlanEdition.STANDARD: dict(monthly_fee=Decimal('100.00')),
+    SoftwarePlanEdition.PRO: dict(monthly_fee=Decimal('500.00')),
+    SoftwarePlanEdition.ADVANCED: dict(monthly_fee=Decimal('1000.00')),
+    SoftwarePlanEdition.ENTERPRISE: dict(monthly_fee=Decimal('0.00')),
+}
+
+BOOTSTRAP_FEATURE_RATES = {
+    SoftwarePlanEdition.COMMUNITY: {
+        FeatureType.USER: dict(monthly_limit=50, per_excess_fee=Decimal('1.00')),
+        FeatureType.SMS: dict(monthly_limit=0),
+    },
+    SoftwarePlanEdition.STANDARD: {
+        FeatureType.USER: dict(monthly_limit=100, per_excess_fee=Decimal('1.00')),
+        FeatureType.SMS: dict(monthly_limit=100),
+    },
+    SoftwarePlanEdition.PRO: {
+        FeatureType.USER: dict(monthly_limit=500, per_excess_fee=Decimal('1.00')),
+        FeatureType.SMS: dict(monthly_limit=500),
+    },
+    SoftwarePlanEdition.ADVANCED: {
+        FeatureType.USER: dict(monthly_limit=1000, per_excess_fee=Decimal('1.00')),
+        FeatureType.SMS: dict(monthly_limit=1000),
+    },
+    SoftwarePlanEdition.ENTERPRISE: {
+        FeatureType.USER: dict(monthly_limit=UNLIMITED_FEATURE_USAGE, per_excess_fee=Decimal('0.00')),
+        FeatureType.SMS: dict(monthly_limit=UNLIMITED_FEATURE_USAGE),
+    },
+}
+
+BOOTSTRAP_FEATURE_RATES_FOR_TESTING = {
+    SoftwarePlanEdition.COMMUNITY: {
+        FeatureType.USER: dict(monthly_limit=2, per_excess_fee=Decimal('1.00')),
+        FeatureType.SMS: dict(monthly_limit=0),
+    },
+    SoftwarePlanEdition.STANDARD: {
+        FeatureType.USER: dict(monthly_limit=4, per_excess_fee=Decimal('1.00')),
+        FeatureType.SMS: dict(monthly_limit=3),
+    },
+    SoftwarePlanEdition.PRO: {
+        FeatureType.USER: dict(monthly_limit=6, per_excess_fee=Decimal('1.00')),
+        FeatureType.SMS: dict(monthly_limit=5),
+    },
+    SoftwarePlanEdition.ADVANCED: {
+        FeatureType.USER: dict(monthly_limit=8, per_excess_fee=Decimal('1.00')),
+        FeatureType.SMS: dict(monthly_limit=7),
+    },
+    SoftwarePlanEdition.ENTERPRISE: {
+        FeatureType.USER: dict(monthly_limit=UNLIMITED_FEATURE_USAGE, per_excess_fee=Decimal('0.00')),
+        FeatureType.SMS: dict(monthly_limit=UNLIMITED_FEATURE_USAGE),
+    },
+}
```

Comment:
I'm gradually working to refactor away repeated code between these two files.  This PR handles a good amount of it, https://github.com/dimagi/commcare-hq/pull/12126 should handle most of the rest